src/pages/public_registration/LogPanel.tsx

```diff
@@ -41,10 +56,39 @@ export const LogPanel = ({ tickets, registration }: LogPanelProps) => {
     cacheTime: 1_800_000, // 30 minutes
   });
 
+  const logs: LogList = [...tickets];
+
+  if (registration.publishedDate) {
+    const registrationPublished: NonTicketLogEntry = {
+      modifiedDate: registration.publishedDate,
+      description: t('registration.status.PUBLISHED_METADATA'),
+    };
+    logs.push(registrationPublished);
+  }
+
+  if (registration.publishedDate && registration.publishedDate < registration.modifiedDate) {
+    const registrationLastModified: NonTicketLogEntry = {
+      modifiedDate: registration.modifiedDate,
+      description: t('common.last_modified'),
+    };
+    logs.push(registrationLastModified);
+  }
+
+  const closedPublishingRequests = tickets.filter(
+    (ticket) => ticket.type === 'PublishingRequest' && ticket.status === 'Closed'
+  );
+  closedPublishingRequests.forEach((ticket) => {
+    const fileRejected: NonTicketLogEntry = {
+      modifiedDate: ticket.modifiedDate,
+      description: t('my_page.messages.files_rejected'),
+    };
+    logs.push(fileRejected);
+  });
+
   return (
     <Box sx={{ display: 'flex', flexDirection: 'column', gap: '0.5rem', mt: '0.5rem' }}>
       {registration && (
-        <StyledStatusMessageBox sx={{ bgcolor: 'publishingRequest.main' }}>
+        <StyledStatusMessageBox key="registration-created-log-entry" sx={{ bgcolor: 'publishingRequest.main' }}>
```

Comment:
burde ikke trenge `key` her?
```suggestion
        <StyledStatusMessageBox sx={{ bgcolor: 'publishingRequest.main' }}>
```

Reply:
Konsollen lagde ulyder uten den :/

Reply:
Er ikke det fra noe gammelt som ikke gjelder? `key` skal bare trengs for array (typisk `map`), og ser ut til å gå fint når jeg tar det vekk lokalt her 🤔